tools/swift-serialize-diagnostics/swift-serialize-diagnostics.cpp

```diff
@@ -82,5 +83,16 @@ int main(int argc, char *argv[]) {
     return EXIT_FAILURE;
   }
 
+  // Print out the diagnostics IDs that are available in YAML but not available
+  // in `.def`
+  if (!yaml.unknownIDs.empty()) {
+    std::string removedIDs;
+    llvm::interleave(
+        yaml.unknownIDs, [&](std::string id) { removedIDs += id; },
```

Comment:
Let's just add `id` to `llvm::errs()` directly, no need to use this separate string here.